appdev/index.rst

```diff
@@ -1,19 +1,56 @@
 App development
 ===============
 
-Great!, you'd like to develop an app for Ubuntu Touch. Here are some resources that we hope help you get started. 
+You'd like to develop an app for Ubuntu Touch? Great!
+
+.. figure:: /_static/images/appdev/applist.jpg
+
+Building your first app should be as simple as:
+
+* `Install clickable <https://clickable.bhdouglass.com/en/latest/install.html>`_
+* Plug in your Ubuntu Touch device
+* Run ``clickable create``
+* Choose "QML Only" from the list of app templates
+* Fill out the requested information for this new app
+* When the app has been generated, enter the newly created directory
+* Run ``clickable``
+
+Voilà you have your first app running on Ubuntu Touch!
+
+Overview
+--------
+
+Ubuntu Touch supports two kinds of apps: Firstly, *native apps* have a frontend written in QML or HTML with their behavior defined in plugins in JavaScript, C++, Python, Rust, or Go. Secondly, *web apps* are special containers to run websites in.
```

Comment:
do you have a link?

Reply:
No. I thought we could actually filter for webapp+ and webapps. Looks like I was wrong. 

They are only called that way because of their additional functionality. They provide something useful rather than just a bookmark everyone can do with webber app.

Maybe add a short explanation `[here](https://github.com/ubports/docs.ubports.com/blob/app_dev/appdev/webapp/index.rst#web-apps)`:
"Web applications are hosted online. They can be as simple as a website, like an online news site, or they can distribute content like videos. They can also have a rich user interface or use the WebGL extension to deliver games online.
Webapps that provide additional functionality instead of just being a bookmark in it's own container, are called webapp+."

I just spotted, `[webapp index.rst](https://github.com/ubports/docs.ubports.com/blob/app_dev/appdev/webapp/index.rst#quick-start)` could do with a link to `[webber app](https://open-store.io/app/webber.timsueberkrueb)`.